server/controllers/azuread.js

```diff
@@ -80,6 +89,10 @@ async function azureAdSignInCallback(ctx) {
       },
     });
 
+    if (!userResponse.data.email) {
+      throw new Error('Email address is not set. Please set email property to the Azure AD user.');
+    }
```

Comment:
I lost some time to realize that the email property was not set for the Azure AD User. I add an error message for clarity.

Reply:
It is possible that someone who forgot to specify an email address for Azure AD users has submitted a Pull Request like this. Usually, the userinfo endpoint is sufficient.

- #34
- #48